enhancements/installer/aws-lb-subnet-selection.md

```diff
@@ -0,0 +1,686 @@
+that it is not exposed to the public Internet."_
+
+When a cluster admin installs a private cluster into a VPC also containing
+public subnets lacking the proper tags, internal LoadBalancer-type services
+may inadvertently use both the public and private subnets, unknowingly introducing
+a security risk (see related [RFE-2816](https://issues.redhat.com/browse/RFE-2816) and
+[OCPBUGS-17432](https://issues.redhat.com/browse/OCPBUGS-17432)).
+
+As a workaround to this type of issue, cluster admins may want to specify the private
+subnet list to ensure that the default IngressController exclusively uses the private
+subnets. See [Setting IngressController Subnets during Installation Workflow](#setting-ingresscontroller-subnets-during-installation-workflow)
+for the workflow for this user story.
+
+#### ROSA HCP: 1 MachinePool in 1 Availability Zone
+
+_"As a user of ROSA HCP, I want to install a cluster with 1 MachinePool in 1
+Availability Zone and have my default IngressController only mapped to the subnets
+in that Availability Zone."_
+
+Similar to [Day 1 IngressController Subnet Selection on AWS](#day-1-ingress-controller-subnet-selection-on-aws).
+
+When a cluster is installed into a VPC containing other subnets in different
+AZs than the cluster subnets, the default IngressController may map to subnets
+in the AZs that are not part of the cluster. This creates a security risk
+because the load balancer is mapping to subnets that may belong to other clusters. This
+user story is derived from [OCM-5969](https://issues.redhat.com/browse/OCM-5969).
+
+See [Setting IngressController Subnets during Installation Workflow](#setting-ingresscontroller-subnets-during-installation-workflow)
+for the workflow for this user story.
+
+#### ROSA HCP: Adding a 2nd MachinePool
+
+_"As a user of ROSA HCP, I want to install a cluster with 1 MachinePool in 1
+AZ, then add a 2nd MachinePool in another AZ and update my IngressController's
+subnets to map to the new AZ."_
+
+A user installed a ROSA HCP cluster with 1 MachinePool in one AZ similar to
+[ROSA HCP: 1 MachinePool in 1 Availability Zone](#rosa-hcp-1-machinepool-in-1-availability-zone),
+but now they want to add another machine pool in a different AZ. The user
+would need to adjust the existing default IngressController subnets by following
+the workflow [Updating an existing IngressController with new Subnets Workflow](/enhancements/ingress/lb-subnet-selection-aws.md#updating-an-existing-ingresscontroller-with-new-subnets-workflow).
+
+Additionally, if the user wants all future IngressController to the new subnet by default, see
+[Updating the Default Subnets for New IngressControllers Workflow](#updating-the-default-subnets-for-new-ingresscontrollers-workflow).
+
+#### Dedicated Ingress Subnets for Security
+
+_"As a cluster admin, I want to install a cluster with dedicated IngressController
+subnets in order to have load balancer VIPs on a dedicated VLAN for applying ACLs."_
```

Comment:
Does this apply only to non-default (custom) ingress controllers?  If so, please specify.